CJBCheatsMenu/Framework/CheatManager.cs

```diff
@@ -109,6 +111,9 @@ internal class CheatManager
         /// <summary>Enables infinite hay.</summary>
         public ICheat InfiniteHay { get; } = new InfiniteHayCheat();
 
+        /// <summary>Prevents Animal friendships from decaying.</summary>
+        public ICheat NoAnimalFriendshipDecay => this.NoAnimalFriendshipDecayImpl;
```

Comment:
There's no need for the backing field, since we can just do this:

```cs
public ICheat NoAnimalFriendshipDecay { get; } = new NoAnimalFriendshipDecayCheat();
```